### PR TITLE
dvc 2.0.18

### PR DESCRIPTION
**Pull request by efiop**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/dvc.rb [hunk 1]
```diff
@@ -423,10 +423,9 @@ class Dvc < Formula
     sha256 "2d475327684562c3a96cc71adf7dc8c4f0565175cf86b6d7a404ff4c771f15f0"
   end
 
-  # This is currently pinned to <3.99 in setup.py. Double check setup.py before updating.
```
Comment (efiop) [inline Formula/dvc.rb:426]: For the record: No longer pinned, so I've removed this comment.

**Review by carlocab — APPROVED** [review]
Thanks, @efiop.

Is `bump-formula-pr` still not working for you?

**Comment by efiop** [comment]
@carlocab I think it wasn't working because of that comment that've removed above. Probably update-python-resources's regex didn't expect comments like that or something, I didn't dig too deeply. Once I've removed that comment, update-python-resources started working again.

**Comment by carlocab** [comment]
Good to know. I should've probably put that comment before the first `resource` so `update-python-resources` could keep working. Not a problem now I guess.

**Pull request closed without merging**